frameworks/Crystal/moonshine/server-redis.cr

```diff
@@ -29,6 +29,11 @@ end
 private def randomWorld
   id = rand(1..ID_MAXIMUM)
   num = REDIS.get("world:" + id.to_s)
+
+  if num == Nil
+    num = 7777
+  end
```

Comment:
Actual problem is this guy started his own branch from a snapshot where I accidentally merged.